lib/kele.rb

```diff
@@ -15,13 +15,37 @@ def initialize(email, pass)
   end
 
   def get_me
-    response = self.class.get('/users/me', headers: { authorization: @auth_token })
+    response = self.class.get('/users/me', headers: auth_header)
     JSON.parse(response.body)
   end
 
   def get_mentor_availability(mentor_id)
-    response = self.class.get("/mentors/#{mentor_id}/student_availability", headers: { authorization: @auth_token})
+    response = self.class.get("/mentors/#{mentor_id}/student_availability", headers: auth_header)
```

Comment:
Yes, we can chat about removing duplication in this... See how lines 23 and 18 are pretty similar? We can extract into a private method that takes the urls and headers as arguments. I can show you on Tuesday!